dlpar/dlpar_main.py

```diff
@@ -631,6 +631,11 @@ def test_offline_cpu_persistence(self):
                         "CPU is still offline even after"
                         "proc operation,Test passed.")
 
+        # Setting cpu online back after TC complete
+        set_cpu1_online = process.system_output(
+            'echo 1 > /sys/devices/system/cpu/cpu1/online',
+            shell=True, ignore_status=False)
```

Comment:
@yeswanth6096 why are you capturing the result in the variable? I don't see any other action performed on this variable